release-notes/opensearch-release-notes-1.0.0.md

```diff
@@ -0,0 +1,510 @@
+
+
+### Opensearch Dashboards Notebooks
+* Add sample notebooks ([#50](https://github.com/opensearch-project/dashboards-notebooks/pull/50))
+* Memoize QueryDataGrid to Reduce Re-renders ([#49](https://github.com/opensearch-project/dashboards-notebooks/pull/49))
+* Update placeholder for search bar to "Search notebook name" ([#42](https://github.com/opensearch-project/dashboards-notebooks/pull/42))
+* Add output only parameter to Context Menu Base URL ([#18](https://github.com/opensearch-project/dashboards-notebooks/pull/18))
+* Add Query Parameter for Selected View ([#17](https://github.com/opensearch-project/dashboards-notebooks/pull/17))
+
+
+### Opensearch Dashboards Reports
+* Use output_only parameter for notebook reports ([#32](https://github.com/opensearch-project/dashboards-reports/pull/32))
+* Add Logic to Auto-populate Notebooks from Context menu ([#30](https://github.com/opensearch-project/dashboards-reports/pull/30))
+* [Query Builder] Correctly handle matched phrases when a single value is specified or when the match phrases is negated ([#33](https://github.com/opensearch-project/dashboards-reports/pull/33))
+* Replace osd header in context menu ([#37](https://github.com/opensearch-project/dashboards-reports/pull/37))
+* Remove visualization editor in visualization reports ([#50](https://github.com/opensearch-project/dashboards-reports/pull/50))
+* Increase chromium timeout to 100s ([#58](https://github.com/opensearch-project/dashboards-reports/pull/58))
+* configure index setting to have default 1 replica and upper bound 2 ([#52](https://github.com/opensearch-project/dashboards-reports/pull/52))
+* Add i18n translation support ([#82](https://github.com/opensearch-project/dashboards-reports/pull/82))
+* PDF report is no more a screenshot, increasing the overall quality ([#82](https://github.com/opensearch-project/dashboards-reports/pull/82))
+* Change Delivery Request Body for Notifications ([#85](https://github.com/opensearch-project/dashboards-reports/pull/85))
+* Remove legacy notifications/delivery related code ([#94](https://github.com/opensearch-project/dashboards-reports/pull/94))
+
+
+### Opensearch Index Management
+* Improve integration with data streams ([#13](https://github.com/opensearch-project/index-management/pull/13))
+* Skip rollover on non-write indices ([#88](https://github.com/opensearch-project/index-management/pull/88))
+
+
+### Opensearch Index Management Dashboards Plugin
+* Support edit transformation name ([#27](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/27))
+
+
+### Opensearch k-NN
+* Change mode for jni arrays release to prevent unnecessary copy backs ([#317](https://github.com/opendistro-for-elasticsearch/k-NN/pull/317))
+* Update minimum score to 0. ([#318](https://github.com/opendistro-for-elasticsearch/k-NN/pull/318))
+* Expose getValue method from KNNScriptDocValues ([#339](https://github.com/opendistro-for-elasticsearch/k-NN/pull/339))
```

Comment:
These point to the old k-NN repo, should they point to https://github.com/opensearch-project/k-NN?

Reply:
Looked into the paths, this seems like an artifact of work that was done in these repros.  @peterzhuamazon Can someone from the k-NN team confirm this?

Reply:
It has already confirmed by k-NN we do have links from both repo since they started dev of k-NN on ODFE until recently move to opensearch.

Reply:
@jmazanec15 approved for k-NN, perfect